include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -324,5 +324,20 @@ def SliceEncodingAttr : DistributedEncoding<"SliceEncoding"> {
   let extraClassDeclaration = extraBaseClassDeclaration;
 }
 
+def DotOperandEncodingAttr : DistributedEncoding<"DotOperandEncoding"> {
```

Comment:
Can you add some comments or embed it into the description to help elaborate the meaning of `DotOperandEncodingAttr`?

This may affect Dot codegen part, cc @Superjomn @goostavz  also.

Reply:
This is the proposal from https://github.com/openai/triton-mlir/discussions/104#discussion-4366348, and will certainly affect dot codegen. I agree that we definitely need heavy comments here and well-defined specs.

Reply:
The original discussion link 
https://github.com/openai/triton-mlir/discussions/104#discussion-4366348
looks already dead...